### PR TITLE
Upgrade the application to dotnet core 2.1

### PR DESCRIPTION
**Pull request by andrelashley**

### CODE REVIEW

File: Gov.News.Media.Website/Program.cs [hunk 1]
```diff
@@ -2,14 +2,20 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.Extensions.Logging;
+using Microsoft.AspNetCore;
```
Comment (bcalvignac) [inline Gov.News.Media.Website/Program.cs]: Please sort the headers

File: Gov.News.Media.Website/Gov.News.Media.Website.csproj [hunk 2]
```diff
@@ -22,7 +22,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.5" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
```
Comment (bcalvignac) [inline Gov.News.Media.Website/Gov.News.Media.Website.csproj]: Add version 2.1.3

File: Gov.News.Media.Website/Gov.News.Media.Website.csproj [hunk 3]
```diff
@@ -22,7 +22,7 @@
...
     <PackageReference Include="Microsoft.AspNetCore.HealthChecks" Version="1.0.0" />
```
Comment (bcalvignac) [inline Gov.News.Media.Website/Gov.News.Media.Website.csproj]: Still 1.0?

Comment (andrelashley) [reply Gov.News.Media.Website/Gov.News.Media.Website.csproj]: Yep, just one version available: https://www.nuget.org/packages/Microsoft.AspNetCore.HealthChecks/

**Pull request merged**